requirements.txt

```diff
@@ -18,31 +18,31 @@ google-generativeai==0.5.0 # for vertex ai calls
 async_generator==1.10.0 # for async ollama calls
 langfuse==2.45.0 # for langfuse self-hosted logging
 prometheus_client==0.20.0 # for /metrics endpoint on proxy
-orjson==3.9.15 # fast /embedding responses
+orjson==3.10.7 # fast /embedding responses
 apscheduler==3.10.4 # for resetting budget in background 
 fastapi-sso==0.10.0 # admin UI, SSO
-pyjwt[crypto]==2.8.0
+pyjwt[crypto]==2.9.0
 python-multipart==0.0.9 # admin UI
-Pillow==10.3.0
+pillow==10.4.0
 azure-ai-contentsafety==1.0.0 # for azure content safety
 azure-identity==1.16.1 # for azure content safety
 opentelemetry-api==1.25.0
 opentelemetry-sdk==1.25.0
 opentelemetry-exporter-otlp==1.25.0
 detect-secrets==1.5.0 # Enterprise - secret detection / masking in LLM requests
-cryptography==42.0.7
+cryptography==43.0.1
 
 ### LITELLM PACKAGE DEPENDENCIES
 python-dotenv==1.0.0 # for env 
 tiktoken==0.7.0 # for calculating usage
 importlib-metadata==6.8.0 # for random utils
-tokenizers==0.14.0 # for calculating usage
+tokenizers==0.20.0 # for calculating usage
 click==8.1.7 # for proxy cli 
 jinja2==3.1.4 # for prompt templates
-certifi==2024.7.4 # [TODO] clean up 
+certifi==2024.8.30 # [TODO] clean up 
 aiohttp==3.10.2 # for network calls
 aioboto3==12.3.0 # for async sagemaker calls
 tenacity==8.2.3  # for retrying requests, when litellm.num_retries set
-pydantic==2.7.1 # proxy + openai req.
-jsonschema==4.22.0 # validating json schema
+pydantic[email]==2.9.1 # proxy + openai req.
```

Comment:
why is this `pydantic[email]` ? @Jacobh2 ?

Reply:
It came from the `pip-tools` which I used to actually find the versions which work together, that it wanted to change to this. Are you perhaps doing any [Email validation](https://docs.pydantic.dev/latest/install/#optional-dependencies)? If not, we can remove it